### ISSUE TITLE
Lack of feedback on flagged stories

### ISSUE THREAD

**Issue by BigPeteB** [fabd/kanji-koohii#156]
After you flag a story, there's no way to tell by looking at the list of stories which ones you've already flagged. You get some visual feedback for a few seconds, but then the ones you've flagged look the same as the ones you haven't. It would be nicer if it was a lasting effect: any time you visit the page, stories you've flagged would have a red "!" or some other indication that you yourself have flagged it. The only way you can tell is to try flagging it and see if the count goes up or down.

Also, a nice bonus would be if stories that you've flagged were automatically hidden when you load the page, regardless of whether they have enough flags to be hidden ordinarily. (Why show a user stories that they've already flagged, thus indicating that they don't like the story?) (It also appears that you can't both flag and favorite a story: the favorite erases your flag. I _guess_ that makes sense, although I'm not sure if it's the behavior I would have picked, but I'm not sure I can make a convincing argument for why someone would want to do both.)

**Comment by fabd** [comment]
Non-trivial change that requires additional db queries per page, refactoring the templateing of shared stories with client side component, and redoing the way shared stories are cached, since the display can vary from user to user.

I'll get to this eventually. At very least needs upgrade to Webpack 4 (too much legacy code).
